Client/mods/deathmatch/logic/CClientVectorGraphicDisplay.h

```diff
@@ -0,0 +1,65 @@
+/*****************************************************************************
+ *
+ *  PROJECT:     Multi Theft Auto v1.0
+ *               (Shared logic for modifications)
+ *  LICENSE:     See LICENSE in the top level directory
+ *  FILE:        mods/shared_logic/CClientVectorGraphicDisplay.h
+ *  PURPOSE:     Client text display base class header
+ *
+ *****************************************************************************/
+
+class CClientVectorGraphicDisplay;
+
+#pragma once
+
+#include "CClientDisplay.h"
+#include "CClientDisplayManager.h"
+
+class CClientVectorGraphicDisplay : public CClientDisplay
+{
+    friend class CClientDisplayManager;
+
+public:
+    CClientVectorGraphicDisplay(CClientDisplayManager* pDisplayManager, CClientVectorGraphic* pVectorGraphic, int ID = 0xFFFFFFFF);
```

Comment:
`ID = 0xFFFFFFFF` should have some form of constant name

Reply:
I've set it to `DISPLAY_VECTORGRAPHIC` - it made the most sense.

https://github.com/multitheftauto/mtasa-blue/blob/master/Client/mods/deathmatch/logic/CClientTextDisplay.h will need the `ID` updating to `DISPLAY_TEXT` in a separate PR.